sources/web/datalab/common.d.ts

```diff
@@ -14,6 +14,14 @@
 
 declare module common {
 
+  interface UserSettings {
```

Comment:
It used to be a little confusing before your rename, I think it's clear now.
Also from the user's perspective, there's only one set of settings they know about, which is user settings, unless they dig really deep to find what we call app settings.